comp/updater/updater/component.go

```diff
@@ -0,0 +1,18 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package updater is the updater component.
+package updater
```

Comment:
What about renamining the component top folder to fleet management? or something of those lines. I think having `updater` by duplicated is a bit confusing 

`comp/fleetmanagment/updater/component.go`
comp/fleetmanagment/api/component.go`